pynamodb/expressions/condition.py

```diff
@@ -0,0 +1,88 @@
+from pynamodb.expressions.util import get_value_placeholder, substitute_names
+
+
+class Operand(object):
+
+    def __init__(self, attribute):
+        self.attribute = attribute
+
+    def __eq__(self, other):
+        return self._compare('=', other)
+
+    def __lt__(self, other):
+        return self._compare('<', other)
+
+    def __le__(self, other):
+        return self._compare('<=', other)
+
+    def __gt__(self, other):
+        return self._compare('>', other)
+
+    def __ge__(self, other):
+        return self._compare('>=', other)
+
+    def _compare(self, operator, value):
+        return Condition(self.attribute, operator, value)
+
+    def between(self, value1, value2):
+        # This seemed preferable to other options such as merging value1 <= attribute & attribute <= value2
+        # into one condition expression. DynamoDB only allows a single sort key comparison and having this
+        # work but similar expressions like value1 <= attribute & attribute < value2 fail seems too brittle.
+        return Between(self.attribute, value1, value2)
+
+    def begins_with(self, prefix):
+        return BeginsWith(self.attribute, prefix)
+
+    def startswith(self, prefix):
+        # A 'pythonic' replacement for begins_with to match string behavior (e.g. "foo".startswith("f"))
```

Comment:
This is more pythonic, but I prefer not having two ways of doing things. Would it be possible to just have `startswith` call `BeginsWith(self.attribute, prefix)` and forgo begins_with?